tools/MMS-data_creator.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE IF NOT EXISTS `mms_database`;
+
+USE `mms_database`;
+
+CREATE TABLE `users` (
+  `email` varchar(50) NOT NULL,
+  `password` varchar(30) NOT NULL,
```

Comment:
Is password being encrypted?

Reply:
Paswswords are going to be encrypted in API and then send to the database.